c2corg_api/views/document_associations.py

```diff
@@ -180,6 +180,7 @@ def get_linked_images(document, lang):
                 Association.child_document_id == Image.document_id,
                 Association.parent_document_id == document.document_id)
             ).
+        order_by(desc(Image.date_time)).
```

Comment:
This is where it can be set depending on the doc. type as I said in the first message.